docs/cmdline-opts/ssl-no-ticket.d

```diff
@@ -0,0 +1,10 @@
+Long: ssl-no-ticket
+Help: Disable curl's use of SSL session-ticket rusing(OpensslSSL)
+Protocols: TLS
+Added: 7.58.0
+---
+(OpenSSL) This option tells curl to disable SSL session ticket during the ssl handshake.
+
+for example:
+ curl https://www.example.com/ -v --ssl-session-file /tmp/sess.pem --ssl-no-ticket
```

Comment:
I think this example shows how the options are hard to understand and use. This disables session-tickets and yet it uses a session from a file... that's a strange "disable"!

Reply:
Example description has been perfected.  
There exists two distinct ways to achieve session reuse: session ID and session ticket.  
If the --ssl-session-file option is specified, the session ticket is used by default, but we know that session ticket is cached on the client side, and if we want to debug the server session ID way, We can use the --ssl-no-ticket option to specify.